app/graphql/graphql_devise/mutations/resend_confirmation.rb

```diff
@@ -0,0 +1,33 @@
+module GraphqlDevise
+  module Mutations
+    class ResendConfirmation < Base
+      argument :email,        String, required: true, prepare: ->(email, _) { email.downcase }
+      argument :redirect_url, String, required: true
+      
+      field :authenticable, User, null: false
```

Comment:
`User` model can be anything when used in projects, let's not define this here. Please check [gem routes](https://github.com/graphql-devise/graphql_devise/blob/master/lib/graphql_devise/rails/routes.rb#L56) to have a better insight on how this is working.

@aarona any particular reason to add a message here?

Reply:
I was trying to follow what DTA was doing here. This success message would allow the front end to give the user instructions on what to do next. I've seen Devise even include an `email` variable with a message like "check #{email} for confirmation instructions." If you guys don't want it, I can remove it but I think it would be good to have here because then you can rely on Rails' i18n to provide a language specific instruction for the user.

Reply:
Ok, I think the message can be useful. The authenticable part needs to be changed

Reply:
Got it. I'll remove it.